manim/mobject/mobject.py

```diff
@@ -1183,7 +1183,16 @@ def get_critical_point(self, direction):
         """
         Picture a box bounding the mobject.  Such a box has
         9 'critical points': 4 corners, 4 edge center, the
-        center.  This returns one of them.
+        center. This returns one of them, along the given direction.
+
+        ::
+            sample = Arc(start_angle=PI/7, angle = PI/5)
```

Comment:
```suggestion
        ::

            sample = Arc(start_angle=PI/7, angle = PI/5)
```

Reply:
Thank you, those are exactly the pointers I was looking for! I'll fix the commit later today--but most importantly, I will set up my system so I can actually test the documentation markup before submitting in the future. :)

Reply:
Hey @Gutza, just checking in to see how you're doing with the documentation. Let me know if you've run into any issues, or reach out to any one of us in the `#docs` channel over on our [discord](https://discord.com/channels/581738731934056449/589666750082187302).

Reply:
Hello, @jsonvillanueva, thank you for reaching out! I did try to join the discord channel, but no joy using that link ("_you find yourself in a strange place_"). I did encounter a surprisingly silly issue when trying to build the documentation myself: if I install manim (`pip3 install manim`), it builds the documentation for the public version (which is not fun, since I can't see what my changes look like); if I uninstall manim (`pip3 uninstall manim`) I receive an error (`Could not import extension manim_directive (exception: No module named 'manim')`). Catch-22.

Reply:
It turns out the issue was even sillier than we could've ever imagined: I never knew I had to execute `python3 -m pip install .` after checking out the git repo, per the [core dev help page](https://docs.manim.community/en/latest/installation/for_dev.html). Thanks to **aquabeam** on discord for pointing me in the right direction, and sorry for wasting everyone's time with this!

Reply:
Thanks for the warning on the discord Link. I sort of assumed that it if you weren't already part of the discord, it would take you to create an account and then redirect you to the `#docs` channel... and if you were logged in/already part of the Manim discord, it would just take you. Good to know. We can continue discussion specific to dev-installation there, but I'm sure most devs use [poetry](https://python-poetry.org/docs/#installation) in [their installations ](https://docs.manim.community/en/latest/installation/for_dev.html#manim-installation-for-developers-with-poetry)... this way you can do an editable install (in tandem with `git` version control)

Reply:
Sorry, I hadn't noticed your reply when I published my final comment. I think it would be wise for me to check out the _poetry_ approach before deciding on a way to move forward. Thank you again!

Reply:
You were right, poetry does indeed offer a way more streamlined experience, that's certainly the way to work!